docs/libraries.md

```diff
@@ -1,52 +1,83 @@
 ---
-title: Libraries
-subtitle: Pre-made language bindings that make accessing the EV3 device drivers easy
-excerpt: "If you are looking to write a program that takes advantage of the EV3's motors, sensors, or other native devices, using a language binding is the way to go."
+title: Interface libraries
```

Comment:
I think I like the title as the more simple "Libraries"

The subtitle can make it clear that these libraries are for interfacing with hardware.